app/views/centers/show.html.slim

```diff
@@ -0,0 +1,36 @@
+doctype html
+html
+  head
+    title PPIS Centers
+
+  body
```

Comment:
as explained last night, the layout is already set on the `/app/views/layouts` and should not be repeated on each of your view